iris_grib/tests/integration/load_convert/test_sample_file_loads.py

```diff
@@ -0,0 +1,223 @@
+# Copyright iris-grib contributors
+#
+# This file is part of iris-grib and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Integration tests for grib2 file loading.
+
+These load various files from iris-test-data, and compare the cube with a
+reference CML file, to catch any unexpected changes over time.
+
+"""
+
+# import iris_grib.tests first so that some things can be initialised
+# before importing anything else.
+import iris_grib.tests as tests
+
+from unittest import skipIf
+
+import iris
+
+# Skip out tests that fail because the file coding is not yet supported.
+# NOTE: now *only* used for the file 'GRIB/y_fastest/y_fast.grib2', as the
+# loader does not support X,Y dimensioned data (as opposed to Y,X).
+# TODO: fix this, or remove the test.
+skip_irisgrib_fails = skipIf(
+    True, "Current grib2 loader does not support this test(s)."
+)
+
+_RESULTDIR_PREFIX = ("integration", "load_convert", "sample_file_loads")
+
+
+@tests.skip_data
```

Comment:
We don't typically do this for iris-grib tests as, unlike iris, we *always* download the iris-test data.
That said, maybe it's a good idea to future-proof these tests, should we decide to copy what iris does.

Reply:
I did check for usage within iris-grib and there are lots of existing uses, so I thought it best to keep it.

Reply:
I see you are right, there are some that do use test.skip_data, but not all. We should try to be consistent with this and perhaps always include the `@test.skip_data`